packages/keyring-controller/src/KeyringController.ts

```diff
@@ -1438,19 +1456,13 @@ export class KeyringController extends BaseController<
    * using the given encryption key and salt.
    *
    * @param encryptionKey - Key to unlock the keychain.
-   * @param encryptionSalt - Salt to unlock the keychain.
    * @returns Promise resolving when the operation completes.
    */
-  async submitEncryptionKey(
-    encryptionKey: string,
-    encryptionSalt: string,
```

Comment:
The salt parameter serves an important role still in identifying whether the key is expired. We should keep it, so long as we support caching the key in this way (see here for details: https://github.com/MetaMask/core/pull/5963/files#r2158856115)